pom.xml

```diff
@@ -325,7 +325,7 @@
                   <goal>exec</goal>
                 </goals>
                 <configuration>
-                  <executable>/bin/sh</executable>
+                  <executable>/bin/bash</executable>
```

Comment:
This is necessary because `popd` and `pushd` in `select-dbms.sh` are bash builtins.